foundationdb/build.rs

```diff
@@ -0,0 +1,17 @@
+extern crate foundationdb_gen;
+
+use std::env;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::PathBuf;
+
+fn main() {
+    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
```

Comment:
Could we use `expect` instead of unwrap? It will make troubleshooting much easier.